src/hooks/useAllServices.ts

```diff
@@ -160,6 +161,10 @@ const useAllServices = () => {
             .get<BundleNavigation>(`${getChromeStaticPathname('navigation')}/${fragment}-navigation.json?ts=${Date.now()}`)
             .catch(() => axios.get<BundleNavigation>(`${isBeta() ? '/beta' : ''}/config/chrome/${fragment}-navigation.json?ts=${Date.now()}`))
             .then(handleBundleResponse)
+            .then(async (bundleNav) => ({
```

Comment:
A todo here to rewrite the function into async await. We are now mixing the two and we will have the possible edge case of eating error message within this branch.